DQM/TrackingMonitorClient/plugins/TrackingAnalyzer.cc

```diff
@@ -91,18 +89,16 @@ void TrackingAnalyser::beginJob() { nLumiSecs_ = 0; }
 // -- Begin Run
 //
 void TrackingAnalyser::beginRun(edm::Run const& run, edm::EventSetup const& eSetup) {
-  edm::LogInfo("TrackingAnalyser") << "TrackingAnalyser:: Begining of Run";
+  edm::LogInfo("TrackingAnalyser") << " Begining of Run";
 
-  if (verbose_)
-    std::cout << "[TrackingAnalyser::beginRun]" << std::endl;
   // Check latest Fed cabling and create TrackerMapCreator
-  unsigned long long cacheID = eSetup.get<SiStripFedCablingRcd>().cacheIdentifier();
-  if (m_cacheID_ != cacheID) {
-    m_cacheID_ = cacheID;
-    edm::LogInfo("TrackingAnalyser") << "TrackingAnalyser::beginRun: "
+  if (fedCablingWatcher_.check(eSetup)) {  //this should check if cabling record has changed
```

Comment:
@makortel can you confirm the `esWatcher` mechanism implemented here is equivalent to the old `cacheIdentifier()`in the previous code?

Reply:
It is, but just for getting EventSetup products (as done here) neither is useful. It is preferable to just get the products unconditionally.

Where are the `fedCabling_` and `detCabling_` used, by the way? (I was wondering if it is necessary to get them at `beginRun()` or if later (lumi/event) retrieval would work as well)

(`ESWatcher` and the lower level `cacheIdentifier()` are intended for cases where some heavy-enough computation is done based on the ES products, and it would be better to do those computations only if IOV changes)